src/Chapter2.hs

```diff
@@ -336,7 +349,7 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end xs = take (end - start + 1) (drop start xs)
```

Comment:
That's right, however, you need to have a few checks on the correct input, otherwise the function could return nit what expected on such inputs 👌🏼